metron-deployment/vagrant/full-dev-platform/README.md

```diff
@@ -13,7 +13,9 @@ Getting Started
 The computer used to deploy Apache Metron will need to have the following components installed.
 
  - [Ansible](https://github.com/ansible/ansible) (2.0.0.2 or 2.2.2.0)
+ - [Docker](https://www.docker.com/community-edition)
  - [Vagrant](https://www.vagrantup.com) 1.8.1
+ - [Vagrant Hostmanager Plugin](https://github.com/devopsgroup-io/vagrant-hostmanager) `vagrant plugin install vagrant-hostmanager`
```

Comment:
Required, at this point.  References that we make like `http://node1:8080` will not work without the Hostmanager plugin.  It just updates the host OS's 'hosts' file.  

In this PR, I moved the Hostmanager Plugin reference from the "Deploy Metron" section to the "Preerequisite" section.  I think this makes more sense as it is not something that you need to do everytime that you deploy Metron, just once; like a prereq.